src/vm/runners/cairo_runner.rs

```diff
@@ -969,12 +966,30 @@ impl CairoRunner {
         Ok(())
     }
 
+    /// Intitializes the runner in order to run cairo 1 contract entrypoints
+    /// Initializes builtins & segments
+    /// All builtins are initialized regardless of use following program ordering then starknet ordering
```

Comment:
Is the ordering really relevant or just how it's implemented? I.e., would the ordering change the behavior in a meaningful way?
If it isn't, I wouldn't mention it. If it is, I think it may merit some more details about why.

Reply:
Its not relevant, Ill remove it